azurerm/internal/services/trafficmanager/resource_arm_traffic_manager_profile.go

```diff
@@ -194,7 +212,7 @@ func resourceArmTrafficManagerProfileCreateUpdate(d *schema.ResourceData, meta i
 
 	read, err := client.Get(ctx, resGroup, name)
 	if err != nil {
-		return err
+		return fmt.Errorf("Error reading TrafficManager profile %s (resource group %s): %v", name, resGroup, err)
```

Comment:
Should we update this to `resourceGroup` as elsewhere in the provider?

Reply:
Not a bad idea! done